civihr_employee_portal/views/civihr_employee_portal.views.inc

```diff
@@ -1166,3 +1189,23 @@ function civihr_employee_portal_views_pre_view(&$view, &$display_id, &$args) {
     );
   }
 }
+
+/**
+ * @param mixed $part The $query->where part
+ * @param array $fields An array to store the fields
+ */
+function get_where_fields($part, &$fields) {
```

Comment:
It's fine to keep this function here, but note that all the other helper functions names in this file use camelCase instead of  snake_case, so maybe it would be better to use camelCase here as well for some consistency (I know, the hooks names are all snake_case, but at least we keep consistency with the existing code :man_shrugging: )